src/cformat.jl

```diff
@@ -1,16 +1,80 @@
 formatters = Dict{ Compat.ASCIIString, Function }()
 
-function sprintf1( fmt::Compat.ASCIIString, x )
+@static if VERSION >= v"0.6.0-dev.1671"
```

Comment:
it means code in the not evaluated conditional won't trigger warnings or some errors, which sometimes is useful but other times is like ifdefs in c, can hide issues if you only ever develop on systems where one of the branches is true and never the other